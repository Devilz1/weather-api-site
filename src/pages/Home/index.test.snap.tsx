import React from 'react';
import renderer from 'react-test-renderer';

import {Home} from './index';

describe('Home', () => {
    it('should render', () => {
        const wrapper = renderer.create(<Home setTitle={jest.fn()} />).toJSON();
        expect(wrapper).toMatchSnapshot();
    });
});
