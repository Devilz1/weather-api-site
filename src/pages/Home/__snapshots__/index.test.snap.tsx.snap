// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render 1`] = `
<div
  className="wrapper"
>
  <h1
    className="title"
  >
    FrontStand
  </h1>
  <button
    onClick={[Function]}
  >
    Click for change title
  </button>
</div>
`;
